ooniprobe/View/RunTestViewController.mm

```diff
@@ -0,0 +1,121 @@
+//
+//  RunTestTableViewController.m
+//  ooniprobe
+//
+//  Created by Lorenzo Primiterra on 06/08/17.
+//  Copyright © 2017 Simone Basso. All rights reserved.
+//
+
+#import "RunTestViewController.h"
+
+@interface RunTestViewController ()
+
+@end
+
+@implementation RunTestViewController
+@synthesize testName, testArguments, testDecription;
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(reloadTestUI) name:@"reloadTable" object:nil];
```

Comment:
This basically register a receiver for `@"reloadTable"` that is the notification you emit, among other things, at the end of the test, correct? This means that, when the test ends, the current view would update, isn't it?

Reply:
yes

Reply:
Thanks